src/math/coneFrustum/ConeFrustum.ts

```diff
@@ -311,8 +311,11 @@ export class ConeFrustum {
 
         }
 
-        return attribute.array as Float32Array;
-
+        if (attribute.array instanceof Float32Array) {
```

Comment:
Prefer the following pattern, whithout "else" branch.
```
if (check is wrong) throw error;
return result;
```

This is more readable, and you can chain multiple checks/throws if needed